mllib/src/main/scala/org/apache/spark/mllib/clustering/BisectingKMeansModel.scala

```diff
@@ -109,7 +109,7 @@ class BisectingKMeansModel private[clustering] (
 
   @Since("2.0.0")
   override def save(sc: SparkContext, path: String): Unit = {
-    BisectingKMeansModel.SaveLoadV1_0.save(sc, this, path)
+    BisectingKMeansModel.SaveLoadV2_0.save(sc, this, path)
   }
 
   override protected def formatVersion: String = "1.0"
```

Comment:
Is this `formatVersion` needed to change too?

Reply:
I'd say yes, but actually this is never used. I think we can actually remove this from the trait.

Reply:
Oh, good catch! need change.

Reply:
I changed the ```formatVersion``` to 2.0. There are quite a few files that implement trait ```Saveable``` and have ```formatVersion```.  I don't feel comfortable to change other files for this PR. Maybe I will open a separate jira to remove ```formatVersion``` from trait ```Saveable```?

Reply:
I agree on that, I already have a patch for removing it (moreover this PR can target 2.4, while removal should be done only on master I think). I am submitting it. Thanks.

Reply:
I created #22830 for that, thanks.